sympy/parsing/autolev/_parse_autolev_antlr.py

```diff
@@ -14,12 +15,11 @@
 
 
 def parse_autolev(autolev_code, include_numeric):
-    antlr4 = import_module('antlr4', warn_not_installed=True)
-    if not antlr4:
+    antlr4 = import_module('antlr4')
+    if not antlr4 or version('antlr4-python3-runtime') != '4.10':
```

Comment:
Maybe the runtime checks should just raise a warning, since we don't actually know if future versions will work or not.

Reply:
We don't do that for any other dependencies. Does antlr not have any compatibility guarantees even for a bugfix release?

Does antlr itself not do any checking that it can compatibly deserialise and report an error that can be caught by sympy?

Reply:
I think the answers to both of those questions might be "no".